src/main/java/org/opensearch/sdk/BaseExtensionRestHandler.java

```diff
@@ -127,4 +134,58 @@ protected ExtensionRestResponse createJsonResponse(RestRequest request, RestStat
             return new ExtensionRestResponse(request, status, fieldName + ": " + responseStr);
         }
     }
+
+    /**
+     * Returns a String message of the detail of any unrecognized error occurred
```

Comment:
Suggestion: add a sentence indicating it is intended for use in error messages to be returned to a user.